packages/charts/src/charts/table/table.styles.ts

```diff
@@ -1,18 +1,44 @@
 import styled from 'styled-components';
-import { Typography } from '@keen.io/ui-core';
+import { TableCell, Typography } from '@keen.io/ui-core';
+import { colors } from '@keen.io/colors';
 import { rgba } from 'polished';
-import { Sticky, DragLine, TableCell } from '@keen.io/ui-core';
+
+import { DRAG_CLASS } from './constants';
 
 export const TableContainer = styled.div`
   height: 100%;
   width: 100%;
-  overflow: auto;
+  overflow: scroll;
+  scroll-margin: 0;
```

Comment:
partially supported in Safari [https://developer.mozilla.org/en-US/docs/Web/CSS/scroll-margin]

check if it looks 👌